frontend/src/layout/navigation-3000/navigationLogic.tsx

```diff
@@ -321,13 +321,13 @@ export const navigation3000Logic = kea<navigation3000LogicType>([
                         },
                         {
                             identifier: Scene.Events,
-                            label: 'Event explorer',
+                            label: 'Events',
                             icon: <IconLive />,
                             to: urls.events(),
                         },
                         {
                             identifier: Scene.DataManagement,
-                            label: 'Data management',
+                            label: 'Data',
```

Comment:
I think the other changes are great, we can't go with this though. There's no data under "Data" here – that lives in "Events" and "People". "Data management" is just about the _shape_ of data. Hence I propose we rename it to the industry-standard "Taxonomy" (though we've got a lot of work left in that regard), and move it below both "People". Or just move while keeping "Data management".

Reply:
I like "Taxonomy".

If we were worried about too much change at once, we keep "Data management" for a while and move it as you suggest, then change it to Taxonomy at a later date, maybe alongside some improvements.

Reply:
It's not just taxonomy though 🤔

Reply:
Moving away from "Data" would avoid confusion with "Data warehouse" down the line, too.

Reply:
Tried something different (see screenshot)...

Reply:
What's the pragmatic approach here? I think data sufficiently gets the point across, even if there is technically data in other places.